ducktape/command_line/main.py

```diff
@@ -165,6 +165,11 @@ def main():
             print("    " + str(test))
         sys.exit(0)
 
+    if args_dict["collect_num_nodes"]:
+        total_nodes = sum(test.expected_cluster_spec.size() for test in tests)
```

Comment:
We can use test.expected_num_nodes here. It's returns the same value right now but would be helpful if there's any change in it in the future.

Reply:
good idea, changed